src/components/EventDetails/EventDetails.js

```diff
@@ -1,12 +1,9 @@
 import React, { Component, PropTypes } from 'react';
 import moment from 'moment';
-import FaFacebookSquare from 'react-icons/lib/fa/facebook-square';
-import FaTwitter from 'react-icons/lib/fa/twitter';
-
 import { DateBlock, Loading } from '../';
 import { eventsAPI } from '../../api';
 import styles from './EventDetails.sass';
-
+import SocialBtns from '../../components/SocialBtns/SocialBtns';
```

Comment:
Make sure to import and export the SocialButns component into ./components/index.js so you can import them together with the other components (see line 3 above)